sql/updates/post-update.sql

```diff
@@ -2,12 +2,13 @@
 DO $$
 DECLARE
  vname regclass;
+ materialized_only bool;
  altercmd text;
  ts_version TEXT;
 BEGIN
     SELECT extversion INTO ts_version FROM pg_extension WHERE extname = 'timescaledb';
 
-    FOR vname IN select format('%I.%I', user_view_schema, user_view_name)::regclass from _timescaledb_catalog.continuous_agg where materialized_only = false
+    FOR vname, materialized_only IN select format('%I.%I', cagg.user_view_schema, cagg.user_view_name)::regclass, cagg.materialized_only from _timescaledb_catalog.continuous_agg cagg
```

Comment:
Not sure I follow what this change has to do with making aggregates fully qualified. Seem unrelated.

Reply:
this rebuilds the view definition so caggs created in earlier versions have fully qualified aggs in view too, we used to only rebuild caggs with realtime aggregation but this changes the code to rebuild any cagg view definitions